modules/cmdb/services/audit/audit.go

```diff
@@ -17,26 +17,24 @@ import (
 	"bytes"
 	"encoding/json"
 	"io"
-	"strconv"
 	"strings"
 	"text/template"
 	"time"
 
-	"github.com/sirupsen/logrus"
-
 	"github.com/erda-project/erda/apistructs"
 	"github.com/erda-project/erda/modules/cmdb/conf"
 	"github.com/erda-project/erda/modules/cmdb/dao"
 	"github.com/erda-project/erda/modules/cmdb/model"
-	"github.com/erda-project/erda/modules/cmdb/utils"
 	"github.com/erda-project/erda/pkg/cron"
 	"github.com/erda-project/erda/pkg/excel"
+	"github.com/erda-project/erda/pkg/ucauth"
+	"github.com/sirupsen/logrus"
 )
 
 // Audit 成员操作封装
```

Comment:
Do not use chinglish

Reply:
I guess what you mean is that there should be no chinese text. and,

that's old stuff, leave it here, we have a plan to refactor all the code. until then, all chinese text will be deleted.